webui/tests/bin/try-for-flakes.sh

```diff
@@ -1,26 +0,0 @@
-#!/bin/bash -e
```

Comment:
keep this one? maybe move it to react dir. it should mostly apply to react unit tests as well for reproducing flakes.

Reply:
yeah that's a good call. I've updated the command to the unit tests so a simple `./bin/try-for-flakes.sh` should run just fine under `react`